python/cudf/cudf/core/udf/groupby_typing.py

```diff
@@ -153,16 +155,29 @@ def caller(data, index, size):
     call_cuda_functions[funcname.lower()][type_key] = caller
 
 
+class GroupUnaryAttrBase(AbstractTemplate):
+    def make_error_string(self, args):
+        fname = self.key.split(".")[-1]
+        return (
+            f"\nSeries.{fname} is not supported for "
+            f"{self.this.group_scalar_type} within JIT GroupBy "
+            "apply.\nPlease file an issue "
+            "requesting support for this feature at cuDF's GitHub page."
```

Comment:
Do we intend to support all possible inputs that can raise this error? It seems like we're asking for a lot of small issues and some of them might even be "wontfix."

Reply:
I think this is a great question and worth discussion. From my point of view, the API is trying to convince the user as best as possible that their function is simply being run in a loop over groups, as in the iterative approach. In an ideal world then, every function we support in JIT groupby apply would support the same set of dtypes that the real `Series.api` supports. We have big holes in that right now, for example `corr` which is `int` only and the rest of the functions which don't support `int16`, `int8` or any unsigned types yet. Each one of these missing overloads is a missing feature IMO and it will take some time to close the gap. 

In raising as such I mainly intend to clarify what is going wrong, however in suggesting the issue I was aiming to create a feedback mechanism that might aid in our prioritization. However maybe as you suggest this is clumsy as we'd be waiting for users to file issues we already know exist.

What would you think of creating an umbrella issue that tracks the gap between supported dtypes and linking to it in the error instead, perhaps suggesting a github reply? Or, if pointing to github seems unnecessary in its entirety I can remove it.

Reply:
A single issue would be better! Maybe you can start it and populate it with known gaps. But generally it’s unusual for cudf to point to GitHub issues in other circumstances such as when calling unsupported keyword arguments from pandas.

Reply:
@shwina What are your thoughts here?

Reply:
I think perhaps pointing a single issue in the error message so folks can provide input as comments is maybe a good compromise? Also it would be a real clickable link in the error message, which is nice. Perhaps something along:

> The `<func>()` function is not supported for the input types `<...>` by cuDF's JIT engine. To see what's currently supported or request support for a new function, go to https://github.com/rapidsai/cudf/issues/42